Jenkinsfile.github

```diff
@@ -86,10 +86,12 @@ pipeline {
 
         stage("Publish") {
             environment {
+                BUILD_ARGS = " "
```

Comment:
You need the build args to contain a space?

Reply:
Not sure - Julian had it this way so I am keeping it rather than changing and needing another testing cycle to verify it can work with no space...